### PR TITLE
Aims 212 background

### PR DESCRIPTION
**Pull request by jaronkk**
If an item metadata sync fails, it needs to be queued to run asynchronously in the background.  

Sync requests made in the foreground will timeout after 3 seconds.  If the foreground request fails it will queue a background request which is not under the same timeout restrictions.

I updated Rails to get ActiveJob to work as expected, since it was failing in rc1.  That caused a schema update when I ran the migrations

This will complete AIMS-212.

### CODE REVIEW

File: app/services/api_get_item_metadata.rb [hunk 1]
```diff
@@ -1,15 +1,33 @@
...
   end
 
   def get_data!
-    ApiHandler.get(:record, barcode: barcode)
+    ApiHandler.get(action: :record, params: { barcode: barcode }, connection_opts: connection_opts)
+  end
+
+  private
+
+  def connection_opts
```
Comment (jgondron) [inline app/services/api_get_item_metadata.rb]: Didn't we want to make these configurable?

Comment (jaronkk) [inline app/services/api_get_item_metadata.rb]: I've moved it to a new file config/settings.yml

**Comment by jgondron** [comment]
:+1:

**Comment by rfox2** [comment]
:+1:

**Pull request merged**